Apps/FCM/ios/FCMSampleApp/MyAppPushNotificationsHandler.swift

```diff
@@ -21,26 +20,12 @@ public class MyAppPushNotificationsHandler : NSObject {
       // They are optional for your setup. 
       config.logLevel = .debug
     }
-    
-    let center  = UNUserNotificationCenter.current()
-    center.delegate = notificationDelegate
+    MessagingPushFCM.initialize(configOptions: nil)    
```

Comment:
same as last PR, can we please add that this is mandatory?

Reply:
After [this conversation is resolved](https://github.com/customerio/customerio-reactnative/pull/225#discussion_r1441769902), I'll act upon this one.